src/Lykke.AzureStorage.Test/TableStorageEntity/AzureTableEntityTests.cs

```diff
@@ -123,7 +123,6 @@ private class ComplexType
         private class TestEntityWithUnknownTypeProperties : AzureTableEntity
         {
             public string StringProperty { get; set; }
-            public ComplexType InvalidTypeProperty { get; set; }
```

Comment:
I would not change DTO classes in this test project. This comment applies to other files in this project.